webclient/core/repository_connector.py

```diff
@@ -672,7 +672,7 @@ def generate_metadata():
             }
             return document_func, headers_func
 
-        if not result:
```

Comment:
Je fakt nešťastné to různé míchání PR. Tohle patří do #3339 a mělo by to zůstat tam, protože kdybychom to z nějakého důvodu neschválili, stejně se nám to do kódu dostane.

Reply:
Ok, vyřešil jsem to pro teď jen změnou base branch na dané PR. Do budoucna se tomu buď vyhněme, nebo to řetěz k tomu nadřazenému PR, aby to bylo jasné, že se to nesmí sloučit dříve...